app/views/reviews/new.html.erb

```diff
@@ -11,7 +11,11 @@
     <h4>Submitted <%= time_ago_in_words(@submission.updated_at) %> ago</h4>
 
     <% unless @submission.times_submitted.nil? %>
-      <h4>Submitted <%= pluralize(@submission.times_submitted, 'time') %></h4>
+      <h4>
+        <span id="times_submitted">Submitted <%= pluralize(@submission.times_submitted, 'time') %></span>
+        (<%= link_to "+", code_review_submission_path(@submission.code_review, @submission, {times_submitted: "increment"}), method: :patch, remote: true %>
```

Comment:
Would `times_submitted: @submission.times_submitted + 1` work? Then in your controller you could just run `@submission.update(submission_params)`.